src/SourceWatcherFuncApp/Entities/DetectorEntity.cs

```diff
@@ -65,5 +65,15 @@ public List<string> AnalysisTypes
                 AnalysisTypesRaw = JsonConvert.SerializeObject(value);
             }
         }
+
+        public string PlatForm { get; set; }
+
+        public string AppType { get; set; }
+
+        public string StackType { get; set; }
+
+        public string HostingEnvironmentType { get; set; }
+
+        public string ResourceProvider { get; set; }
```

Comment:
I dont think you need `StackType` or `HostingEnvironmentType`. You have plans for these?